spec/factories/api/monetary_exchange_rate.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  sequence(:monetary_exchange_rate_id)
+
+  factory :monetary_exchange_rate, class: 'Api::MonetaryExchangeRate' do
+    id { generate(:monetary_exchange_rate_id) }
+    child_monetary_unit_code { 'GBP' }
```

Comment:
`child_monetary_unit_code` does not sound a great naming 🤔 ... but I think it was already there

Reply:
Yeah, this is what was chosen by the EU I believe - not by us.